Rubberduck.Parsing/Symbols/ProceduralModuleDeclaration.cs

```diff
@@ -18,21 +18,12 @@ public sealed class ProceduralModuleDeclaration : Declaration
             : base(
                   qualifiedName,
                   projectDeclaration,
-                  projectDeclaration,
                   name,
-                  null,
-                  false,
-                  false,
-                  Accessibility.Public,
                   DeclarationType.ProceduralModule,
-                  null,
-                  null,
-                  Selection.Home,
-                  false,
-                  null,
                   isUserDefined,
                   annotations,
-                  attributes) { }
+                  attributes)
+        { }
 
         public ProceduralModuleDeclaration(ComModule statics, Declaration parent, QualifiedModuleName module,
```

Comment:
Is `statics` the best name for this parameter, and why does it differ so much from the constructors for `ClassModuleDeclaration`?

Reply:
IIR I picked the name `statics` because a COM module that does not define an interface or a class (`TKIND_MODULE`) is analogous to a `static class` in .NET, and `module` gets confusing due to how it is (over) used in VB parlance.

The ctor is a stripped down version of `ClassModuleDeclaration` because it doesn't carry some of the flags that are only relevant to coclasses and interfaces (e.g. a standard module can't have a default member, can't be a control, can't predeclared, etc.).